share/wake/lib/gcc_wake/pkgconfig.wake

```diff
@@ -51,6 +51,7 @@ target pkgConfigImp args =
     require Some output =
         result.getJobStdout
         | getWhenFail ""
+        | replace `[\r]` ""
```

Comment:
Is the character group for clarity here?  (Which is likely enough of a purpose to leave it in.)
```suggestion
        | replace `\r` ""
```

Reply:
yep, I asked Ashley to put it in there like that.